src/wallet/rpc/spend.cpp

```diff
@@ -1633,7 +1633,7 @@ RPCHelpMan walletcreatefundedpsbt()
         }, true
     );
 
-    UniValue options = request.params[3];
+    UniValue options{request.params[3].isNull() ? UniValue::VOBJ : request.params[3]};
```

Comment:
Is this well-defined, valid C++? At least in normal C, the `?:` operator requires both results to be the same type...

Reply:
Yes, it should be, though it likely involves the copy-constructor. Pretty sure I copied it from the other places where this is used consistently.